src/components/Sidebar/FooterMenu.react.js

```diff
@@ -43,8 +43,10 @@ export default class FooterMenu extends React.Component {
           onExternalClick={() => this.setState({ show: false })}>
           <div className={styles.popup}>
             <a href={`${mountPath}logout`}>Log Out <span className={styles.emoji}>👋</span></a>
-            <a target='_blank' href='https://www.parse.com/docs/server/guide'>Server Guide <span className={styles.emoji}>📚</span></a>
-            <a target='_blank' href='https://www.parse.com/help'>Help <span className={styles.emoji}>💊</span></a>
+            <a target='_blank' href='https://github.com/ParsePlatform/parse-server/wiki'>Server Wiki <span className={styles.emoji}>📚</span></a>
```

Comment:
Can  you link to parseplatform.github.io/docs/parse-server/guide instead? The wiki is no longer hosting the guide.